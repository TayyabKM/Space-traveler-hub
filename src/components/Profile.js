import React from 'react';

export default function Profile() {
  return (
    <div>
      <h1>THIS IS MY PROFILE</h1>
    </div>
  );
}
