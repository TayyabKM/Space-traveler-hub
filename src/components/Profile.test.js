import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Profile from './Profile';

describe('Profile Page', () => {
  it('should render correctly', () => {
    const component = TestRenderer.create(
      <Provider store={store}>
        <Profile />
      </Provider>,
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
