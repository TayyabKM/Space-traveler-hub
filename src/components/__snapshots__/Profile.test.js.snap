// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Page should render correctly 1`] = `
<div
  className="p-2 container"
>
  <div
    className="row row-cols-md-2"
  >
    <div
      className="col"
    >
      <h3>
        My Missions
      </h3>
      <div
        className="fade alert alert-primary show"
        role="alert"
      >
        You haven't joined any missions yet.
      </div>
      <div
        className="list-group"
        onKeyDown={[Function]}
      />
    </div>
    <div
      className="col"
    >
      <h3>
        My Rockets
      </h3>
      <div
        className="fade alert alert-primary show"
        role="alert"
      >
        You haven't reserved any rockets yet.
      </div>
      <div
        className="list-group"
        onKeyDown={[Function]}
      />
    </div>
  </div>
</div>
`;
